packages/pds/src/db/message-queue/index.ts

```diff
@@ -51,6 +56,15 @@ export class SqlMessageQueue implements MessageQueue {
     this.cursorExists = true
   }
 
+  async ensureCaughtUp(): Promise<void> {
+    try {
+      await this.processAll()
+    } catch (err) {
+      log.error({ err }, 'error ensuring queue is up to date')
+    }
+    setTimeout(this.ensureCaughtUp, 60000) // 1 min
```

Comment:
I think that it _might_ need to be like this, to avoid `this` not being bound when the callback is called:
```suggestion
    setTimeout(() => this.ensureCaughtUp, 60000) // 1 min
```

Reply:
ah shit you're totally right